manta-accounting/src/wallet/signer/mod.rs

```diff
@@ -1533,50 +984,15 @@ where
     /// Returns the [`Address`] corresponding to `self`.
     #[inline]
     pub fn address(&mut self) -> Address<C> {
-        let account = self.state.accounts.get_default();
-        account.address(&self.parameters.parameters)
+        methods::address(self.parameters(), self.state.accounts())
```

Comment:
using `&self.parameters` and `&self.state.accounts` that align to other methods? or change other methods to `self.parameters()` and `self.state.accounts()`?